src/server/ua_subscription_alarms_conditions.c

```diff
@@ -2638,4 +2718,175 @@ UA_StatusCode UA_Server_deleteCondition(UA_Server *server, const UA_NodeId condi
     /* Delete from address space */
     return UA_Server_deleteNode(server, condition, true);
 }
-#endif /* UA_ENABLE_SUBSCRIPTIONS_ALARMS_CONDITIONS */
+
+static UA_StatusCode getLowLimit(UA_Server *server, UA_NodeId conditionId, UA_Double *lowLimit) {
```

Comment:
Shorten the lines.
Add a break after `static UA_StatusCode` as is done in the rest of the code.
The same below.